.github/workflows/labeler.yml

```diff
@@ -0,0 +1,72 @@
+# .github/labeler.yml
+
+# Label PRs that modify documentation files
+Documentation:
```

Comment:
Defined clear labeling rules for different directories, which will help maintain organization in our pull requests.